spring-cloud-stream-binder-test/src/main/java/org/springframework/cloud/stream/binder/AbstractBinderTests.java

```diff
@@ -122,7 +122,7 @@ public void testSendAndReceive() throws Exception {
 		// Let the consumer actually bind to the producer before sending a msg
 		binderBindUnbindLatency();
 		moduleOutputChannel.send(message);
-		Message<?> inbound = receive(moduleInputChannel);
+		Message<?> inbound = receive(moduleInputChannel, 5);
```

Comment:
I had this locally to see if it helps avoiding those intermittent test errors we see on the binder tests. will revert this.